src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -61,13 +58,17 @@ export class ManifoldInvoices {
           (edge) => edge.node.id === this.selectedId
         );
 
+        if (!invoice) {
+          return null;
+        }
```

Comment:
This was the other place where `strictNullChecks` saved us from a potentially-missing thing